modules/compression/src/workers/compression-worker.ts

```diff
@@ -0,0 +1,83 @@
+import {createWorker} from '@loaders.gl/worker-utils';
```

Comment:
The node and browser workers are almost identical long files, that doesn't feel too good... Could they share code?

Reply:
I need unified file structure to run workers from src: `moudels/${module}/src/workers/${workerName}-node.ts`

Reply:
Yes... could they import each other or some common file? Or they need to be self contained in a single file?